reader/components/toolbar/Toolbar.scss

```diff
@@ -45,33 +49,30 @@
 
       @include media-breakpoint-down(xs) {
         width: 35px;
-        padding: 0.30rem;
+        padding: 0.3rem;
       }
     }
   }
 
   .page-number-navigation {
-    margin-left: -1rem;
-    margin-right: -0.75rem;
-
     .input-change-page-number,
     .pages-count {
       display: inline-block;
-      width: 35px;
-      text-align: center;
+      width: 2.4rem;
     }
 
     .input-change-page-number {
+      text-align: right;
       background-color: inherit;
-      border: 0;
+      border: 1px solid transparent;
```

Comment:
Consistency. Historically this hack is outdated. I would cause element shifting without `border-box`. However, you will be able to see negative effect of changing border size if have correct transition.

I change colour only.